### PR TITLE
[JN-1289] Kit collection UX (participant)

### PR DESCRIPTION
**Pull request by MatthewBemis**
#### DESCRIPTION (include screenshots, and mobile screenshots for participant UX)

Adds the (currently hidden) participant UX flow for in-person kits. Note the branching in step 3, where a user can select their return option. Since in person kits are not implemented on the backend yet, this utilizes a hardcoded kit ID to show the full flow.

I expect this to be most of the way towards what we want for the final flow, but obviously there are some tweaks that will be made once the full thing is working end-to-end.

Step 1
Screenshot 2024-08-14 at 10 19 14 AM

Step 2 + Step 3
Screenshot 2024-08-14 at 10 19 22 AM

Step 4 (return via mail)
Screenshot 2024-08-14 at 10 20 03 AM

Step 4 (return in person)
Screenshot 2024-08-14 at 10 20 10 AM

Desktop view:
Screenshot 2024-08-14 at 12 33 50 PM

Unconsented view:

Screenshot 2024-08-14 at 11 06 10 AM

#### TO TEST:  *(simple manual steps for confirming core behavior -- used for pre-release checks)*

Log in as basic@test.com and go to https://sandbox.demo.localhost:3001/hub/kitInstructions
Try out the flow and verify that it looks good. 

Log in as newbie@test.com and go to  https://sandbox.demo.localhost:3001/hub/kitInstructions
Confirm you're prompted to complete consent

Try out the flows on mobile and desktop

### CODE REVIEW

File: ui-participant/src/hub/kit/KitInstructions.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
+import React, { lazy, useState } from 'react'
+import { Link } from 'react-router-dom'
+import { useActiveUser } from 'providers/ActiveUserProvider'
+import { KitCollectionStep, KitReturnType } from '@juniper/ui-core'
+import { usePortalEnv } from 'providers/PortalProvider'
+const QRCode = lazy(() => import('react-qr-code'))
```
Comment (devonbush) [inline ui-participant/src/hub/kit/KitInstructions.tsx:6]: good call lazying this

File: ui-participant/src/hub/kit/KitInstructions.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+//TODO: JN-1294, implement i18n for this entire component
+
+export function KitInstructions() {
+  const { ppUser, enrollees } = useActiveUser()
+  const { portalEnv } = usePortalEnv()
+  const activeEnrollee = enrollees.find(enrollee => enrollee.profileId === ppUser?.profileId)
+  const isConsented = activeEnrollee?.consented
+  const [selectedReturnType, setSelectedReturnType] = useState<KitReturnType>()
+  //TODO: JN-1259, replace with an actual kit
```
Comment (devonbush) [inline ui-participant/src/hub/kit/KitInstructions.tsx:16]: concur, I think these PRs are easier to build/review when the i18n is done separately

**Comment by devonbush** [comment]
Looks good, good choice to hide it for now.  Eventually, I imagine we'll have an "enable in-person kits" setting in studyEnvironmentConfig?

**Review by connorlbark — APPROVED** [review]
Looks great! Super minor, but: Should the next step only show up once you've completed the prior step? Either way, I'm excited to try it out in full with a real backend!

**Review by devonbush — APPROVED** [review]

**Comment by MatthewBemis** [comment]
@connorlbark that's a good point, I will address hiding/greying out later steps in a future PR!

**Pull request merged**